lib/filters/sandbox_filter.py

```diff
@@ -0,0 +1,28 @@
+from lib.log_processor import ProcessedLogEntry
+
+
+def is_formal_environment(log_name):
```

Comment:
could this be '_is_formal_environment'? Can't see it being used anywhere else and they are '_is_totalmobile_alert' etc. below (in slack/slack_message.py)

Reply:
Agreed, good sir! 🧐